MediaBrowser.Providers/Music/MusicBrainzAlbumProvider.cs

```diff
@@ -720,82 +720,6 @@ private string GetFirstReleaseGroupId(XmlReader reader)
             return null;
         }
 
-        private long _lastMbzUrlQueryTicks = 0;
```

Comment:
care to explain how this deletion is justified and why it is in a PR removing Firebase?

Reply:
I seem to get what's happening now. There's a few changes in this PR. One is the removal of Firebase, which is covered in:
- `Emby.Server.Implementations/Session/FirebaseSessionController.cs`
- `Emby.Server.Implementations/Session/SessionManager.cs`

Then the other changes are removing a portion of the ApplicationHost file that reads an empty `values.txt` file located here: jellyfin/Emby.Server.Implementations/values.txt (Note that this file is also blank in the upstream Emby, it is likely they have hidden it some time ago and use it to keep alternate URLs for services).

As a result, the fixes in the two metadata providers are to remove the requests to AppHost to read the values, and get alternate URLs if they exist. Since they don't in any copy we have, it's okay to remove these requests, and just hard code in or use the base URLs that currently exist.

While we do eventually want to allow users to provide their own API keys, this values.txt won't be the way to do so, so I think this set is good to go.

@Bond-009 - Is my review correct?

Reply:
@anthonylavado Yes.
as for the reason these two are bundled: firebase was also using the `GetValues` method when I saw it was useless without it I looked into what its function was and I decided to completely remove it.

Reply:
That satisfies me. I’ll mark as resolved and give an explicit approval.